charts/incubator/teslamate/questions.yaml

```diff
@@ -0,0 +1,121 @@
+# Include{groups}
+portals:
+  open:
+# Include{portalLink}
+questions:
+# Include{global}
+# Include{workload}
+# Include{workloadDeployment}
+# Include{replicas1}
+# Include{podSpec}
+# Include{containerMain}
+                                - variable: env
+                                  label: "Image Environment"
+                                  schema:
+                                    additional_attrs: true
+                                    type: dict
+                                    attrs:
+                                      - variable: MQTT_HOST
+                                        label: "Mqtt Host"
+                                        description: "Hostname of the broker."
+                                        schema:
+                                          type: string
+                                          default: ""
+                                      - variable: MQTT_PORT
+                                        label: "Mqtt Port"
+                                        schema:
+                                          type: int
+                                          default: 1883
+                                      - variable: MQTT_HOST
+                                        label: "Mqtt Host"
+                                        description: "Port of the broker."
+                                        schema:
+                                          type: string
+                                          default: ""
```

Comment:
Weird i didnt actually write anything on this commit, i just used the github UI to batch the suggestions and commit them. i'll fix this and commit manually. im adding the option to disable mqtt since its only needed if you want to integrate with home assitant and other things not for the data logging. another commit incomming soon.